### PR TITLE
HDDS-4247. Fixed log4j usage in some places

### PR DESCRIPTION
**Pull request by lamber-ken**
## What changes were proposed in this pull request?

Fixed log4j usage in some places.

Examples, `{}` is redundant.
```
LOG.error("Error while scrubbing pipelines {}", e);
```

## What is the link to the Apache JIRA

https://issues.apache.org/jira/projects/HDDS/issues/HDDS-4247

## How was this patch tested?

not need.

### CODE REVIEW

File: hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/security/OzoneDelegationTokenSecretManager.java [hunk 1]
```diff
@@ -600,7 +600,7 @@ public void run() {
 
         }
       } catch (InterruptedException ie) {
-        LOG.error("ExpiredTokenRemover received {}", ie);
+        LOG.error("ExpiredTokenRemover received.", ie);
```
Comment (adoroszlai) [inline hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/security/OzoneDelegationTokenSecretManager.java:603]:
```suggestion
        LOG.info("ExpiredTokenRemover was interrupted.", ie);
```

**Review by adoroszlai — COMMENTED** [review]
Thanks @lamber-ken for fixing these placeholder issues.  LGTM, but I think the message in `ExpiredTokenRemover` is incomplete.

File: hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/security/OzoneDelegationTokenSecretManager.java (hunk 1 above)
Comment (lamber-ken) [reply hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/security/OzoneDelegationTokenSecretManager.java]: @adoroszlai done, thanks

**Review by adoroszlai — APPROVED** [review]

**Comment by adoroszlai** [comment]
Thanks @lamber-ken for the contribution.

**Pull request merged**